app/client/src/layout/Header.jsx

```diff
@@ -242,8 +244,10 @@ const Header = (props) => {
       sx={{
         borderBottom: 3,
         borderColor: (theme) => theme.palette.secondary.main,
```

Comment:
Instead of a function we can use "secondary.main" as a string and the 'sx' prop should take care of the rest.